docs/api_docs/getting_started_with_apis.rst

```diff
@@ -0,0 +1,39 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _getting_started_with_apis:
+
+***************************************
+Getting Started with Vertiq's APIs
+***************************************
+
+What is the Purpose of Vertiq's APIs
+===========================================
+Vertiq's APIs provide a simple mechanism for communicating with your modules via the IQUART protocol. IQUART is Vertiq's proprietary serial 
+protocol meant to allow configuration and control of all Vertiq modules. To learn more about IQUART, please see :ref:`the protocol's documentation <uart_messaging>`. 
+Our APIs provide support for Python, Matlab, C++, and Arduino. These options provide flexibility to work easily on both PC-based and embedded solutions.
+
+If you intend to use the API specifically in a PC based environment, we recommend using either the Python or Matlab APIs.
+
+If you intend to use the API in an embedded system, we recommend using the C++ or Arduino libraries.
+
+What Can the APIs Do
+========================
+Essentially, each API is a language and hardware specific implementation of the IQUART protocol with access to information about each of the IQUART clients. 
```

Comment:
When you say "hardware specific", what do you mean? Is that referring specifically to the Arduino libraries? The Python, Matlab, and C++ ones are hardware agnostic I would say, and even the "arduino" one works across a pretty broad range of hardwares